components/wb-bubble/wb-bubble-qa-fr.html

```diff
@@ -0,0 +1,35 @@
+
+&lt;p&gt;Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer sapien nulla, laoreet nec tellus eget, iaculis porttitor eros. In consequat convallis nisl, in euismod ligula varius quis. Donec vitae iaculis nunc, vitae posuere risus. Quisque eu felis eget purus ultricies venenatis accumsan sed lorem. Fusce et nisl in diam ultricies vehicula. Nullam leo mi, vehicula ut laoreet quis, convallis non odio. Fusce commodo quam sit amet ipsum sagittis cursus. Ut non sem ac risus tincidunt imperdiet ac quis elit. Duis feugiat blandit tellus, nec aliquam lacus ultricies vitae. Vestibulum aliquet consectetur ligula, a suscipit enim euismod ac. Suspendisse non elit eros. &lt;/p&gt;
+
+&lt;p&gt;Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Sed sodales consectetur nisi id venenatis. Praesent placerat ut nisl sed accumsan. Donec augue elit, accumsan vitae auctor gravida, volutpat nec nisi. Mauris eget aliquet mauris, at dictum massa. Duis sit amet leo in justo varius finibus. Integer placerat magna vel urna sollicitudin, in aliquet tortor pretium. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Integer feugiat, sapien sed vulputate egestas, ipsum massa gravida velit, sit amet consequat mauris justo in risus. Pellentesque dapibus luctus massa nec egestas. Nunc lacinia malesuada tortor, feugiat facilisis sem laoreet hendrerit. Nullam tincidunt enim nibh, vitae lacinia est convallis vitae. Mauris id massa enim. &lt;/p&gt;
+
+&lt;p&gt;Etiam vehicula auctor lacus. In luctus felis enim. Cras consectetur tristique lacinia. Cras urna lorem, aliquam a auctor ut, congue eget ex. Phasellus commodo id mauris ut rutrum. Ut efficitur scelerisque sem, ac condimentum diam congue ac. Maecenas hendrerit, libero sit amet cursus pulvinar, lorem arcu aliquet leo, nec imperdiet leo ipsum id diam. Aliquam erat nisl, volutpat et dictum vitae, congue et enim. Morbi tempor velit ut tortor ullamcorper, et ullamcorper odio fermentum. Donec non pellentesque sem. Donec tincidunt convallis tortor. Donec in nisl consectetur, efficitur lectus scelerisque, dictum augue. Quisque consequat ornare sem, et mollis augue luctus ut. &lt;/p&gt;
+
+&lt;p&gt;Praesent vitae sem ullamcorper, finibus nibh ut, dapibus diam. Curabitur risus sapien, dapibus suscipit auctor at, tristique non quam. Suspendisse quis enim commodo, pulvinar ligula in, lobortis magna. Vestibulum pulvinar a odio non faucibus. Vestibulum ullamcorper condimentum diam, vitae mattis massa porttitor vel. In non tortor libero. Nunc sit amet metus rutrum, rutrum erat aliquam, auctor leo. In nunc enim, ultrices eu sodales vitae, rhoncus eget turpis. Vivamus mattis, lectus eget aliquam pharetra, ex purus blandit erat, et mattis nibh purus nec turpis. Aliquam eu metus nec urna dictum aliquet eget a elit. Duis interdum ipsum eget finibus blandit.&lt;/p&gt;
+
+&lt;a href="#" class="wb-bubble"&gt;Lien rapide&lt;/a&gt;
```

Comment:
I would only put this line alone in the code block, and move it above the Lorem ipsum on the page so that it pops out.